dnf/cli/commands/__init__.py

```diff
@@ -370,9 +370,9 @@ def run_on_repo(self):
                 for pkg_spec in self.opts.pkg_specs:
                     try:
                         self.base.install(pkg_spec, self.reponame)
-                    except dnf.exceptions.MarkingError:
-                        msg = _('No package %s available.')
-                        logger.info(msg, self.output.term.bold(pkg_spec))
+                    except dnf.exceptions.MarkingError as e:
+                        msg = '{}: {}'.format(e.value, self.base.output.term.bold(pkg_spec))
```

Comment:
Seems this formatting with bold is used often enough to warrant putting it into a `MarkingError` method? Name suggestion :slightly_smiling_face:  `str_term_highlight`